### ISSUE TITLE
False positive for `no-commented-out-code` with `type` and `interface` in TypeScript

### ISSUE THREAD

**Issue by Arttse** [cartant/eslint-plugin-etc#32]
For example:
```ts
interface MyExample {
  // City
  a: string;
  /**
   * City
   */
  a1: string;
  // Город
  b: string;
  // 町
  c: string;
  // İlçe
  d: string;
  // Քաղաք
  e: string;
}
```

---

Plugin version `1.5.2`

**Comment by cartant** [comment]
There was a `continue` in the wrong place, that's all. Should be fixed in 1.5.3.
